src/layer_values_monitor/custom_feeds.py

```diff
@@ -0,0 +1,178 @@
+"""Custom price feeds for tokens not supported by telliot-feeds."""
```

Comment:
I think its better for consistency to add these feeds to telliot to keep it in line with the other ones.